File: WebApplication/Models/Course.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebApplication.Models
{
    public class Course : EntityBase<Group>
    {
        public string Name { get; set; }
        //public List<GroupCourse> GroupCourses { get; set; }
    }
}

File: SparkApplication/src/main/java/Models/Course.java
package Models;

public class Course extends  EntityBase{
    private String name;

    public Course(){
        super();
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return getId() + "," + getType() + "," + name;
    }
}
